index.bs

```diff
@@ -650,11 +658,12 @@ Registry instances are initially created with the internal slots described in th
 1. Let _hook_ be GetMethod(_loader_, @@fetch).
 1. // TODO: metadata object
 1. Let _p0_ be the result of promise-calling _hook_(_key_).
-1. Let _p_ be the result of transforming _p0_ with a fulfillment handler that, when called with argument _payload_, runs the following steps:
+1. Let _p1_ be the result of transforming _p0_ with a new pass-through promise.
```

Comment:
`p1` is now the wrapping promise that can be resolved via `p0` or via `.provide()`, no matter which way it gets resolved, we will always update the state.